net/beanstalkd/patches/902-replace-posix_fallocate.patch

```diff
@@ -0,0 +1,18 @@
+--- a/linux.c
++++ b/linux.c
+@@ -1,4 +1,6 @@
+ #define _XOPEN_SOURCE 600
++#include <unistd.h>
++#include <sys/types.h>
+ #include <stdint.h>
+ #include <fcntl.h>
+ #include <stdlib.h>
+@@ -19,7 +19,7 @@ static int epfd;
+ int
+ rawfalloc(int fd, int len)
+ {
+-    return posix_fallocate(fd, 0, len);
++    return ftruncate(fd, len);
```

Comment:
Just a suggestion: in order to avoid the need for carrying a patch file you could also use something like `TARGET_CFLAGS+=$(if $(CONFIG_USE_MUSL),-D"posix_fallocate(fd,off,len)=ftruncate(fd,len)")`